src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,214 @@
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String query = "INSERT INTO cars (model, manufacturer_id) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement saveCarStatement = connection.prepareStatement(query,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            saveCarStatement.setString(1, car.getModel());
+            saveCarStatement.setLong(2, car.getManufacturer().getId());
+            saveCarStatement.executeUpdate();
+            ResultSet resultSet = saveCarStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                Long id = resultSet.getObject(1, Long.class);
+                car.setId(id);
+            }
+            return car;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't create "
+                    + car + ". ", e);
+        }
+    }
+
+    @Override
+    public Car get(Long id) {
+        String query = "SELECT cars.id, cars.model, cars.manufacturer_id, "
+                + "manufacturers.name, manufacturers.country "
+                + "FROM cars JOIN manufacturers ON cars.manufacturer_id = manufacturers.id"
+                + " WHERE cars.id = (?) AND is_deleted = false";
+        Car car = null;
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getCarStatement = connection.prepareStatement(query)) {
+            getCarStatement.setLong(1, id);
+            ResultSet resultSet = getCarStatement.executeQuery();
+            if (resultSet.next()) {
+                car = getCar(resultSet);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get car by id " + id, e);
+        }
+        if (car != null) {
+            car.setDrivers(getAllDriversForCar(id));
+        }
+        return car;
+    }
+
+    private List<Driver> getAllDriversForCar(Long carId) {
+        String query = "SELECT drivers.id, name, license_number FROM drivers "
+                + "JOIN cars ON drivers.id = cars.id WHERE cars.id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement getAllDriverStatement
+                         = connection.prepareStatement(query)) {
+            getAllDriverStatement.setLong(1, carId);
+            ResultSet resultSet = getAllDriverStatement.executeQuery();
+            List<Driver> driverList = new ArrayList<>();
+            while (resultSet.next()) {
+                driverList.add(parseDriversFromResultSet(resultSet));
+            }
+            return driverList;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get a list of car by driver id "
+                    + carId, e);
+        }
+    }
+
+    private Driver parseDriversFromResultSet(ResultSet resultSet) throws SQLException {
```

Comment:
```suggestion
    private Driver parseDriverFromResultSet(ResultSet resultSet) throws SQLException {
```
you are returning only one driver, not a list of drivers